package.json

```diff
@@ -31,14 +31,14 @@
     "@nextcloud/axios": "^2.4.0",
     "@nextcloud/calendar-js": "6.1.0",
     "@nextcloud/cdav-library": "1.1.0",
-    "@nextcloud/dialogs": "^5.1.1",
+    "@nextcloud/dialogs": "4.1.0",
```

Comment:
Have you found issues using v5 with a vue app? Would be interested :)

Reply:
No, there are no issues. The downgrade is just necessary at the moment because since version `4.2.0` `nextcloud/dialogs` depends on `nextcloud/vue@8` and `vue2`, so it needs to be migrated to vue 3 first. But that is not an issues here, because the Tasks app only uses `showError` from the lib which works the same in `4.1.0`.